### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by VioMNE**

### CODE REVIEW

File: javascript/javascript1/week2/h2.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      console.log('Please enter a name');
+  } else if (class07Students.includes(studentName)) {
+      console.log('Student' + studentName + 'is already in the class');
+  } else if (studentName === 'Queen' || class07Students.length < 6) {
+      class07Students.push(studentName);
+  } else if (class07Students.length >= 6) {
+      console.log('Cannot add more students to class');
+  } else {
+      class07Students.push(studentName);
+  }
```
Comment (cecastosic) [inline javascript/javascript1/week2/h2.js:97]: what should be returned at the end?

File: javascript/javascript1/week2/h2.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+addStudentToClass();
+addStudentToClass('Violeta');
+addStudentToClass('Marija');
+addStudentToClass('Milos');
+addStudentToClass('Mirko');
+addStudentToClass('Petar');
+addStudentToClass('Jelna');
```
Comment (cecastosic) [inline javascript/javascript1/week2/h2.js:106]:
```suggestion
addStudentToClass('Jelena');
```

Comment (cecastosic) [reply javascript/javascript1/week2/h2.js:106]: Nice names 🙂

**Review by cecastosic — COMMENTED** [review]
Nice job @VioMNE 👍 Please check my comments. It would be nice if you can try using Template literals and format your code. Please choose do you want to use `""` or `''`.  Good reading about return https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return 🌸

File: javascript/javascript1/week2/h2.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
+//Flight booking fullname function
+// Created a function called getFullname that returns a fullname with two parameters
+
+function getFullname(firstname, surname) {
+    return firstname + " " + surname;
+
```
[the remark this answers is not in the document]
Comment (VioMNE) [reply javascript/javascript1/week2/h2.js:6]: Its says that pretter is on but it doesnt do much, i have to figure it out. :)

**Comment by VioMNE** [comment]
> Nice job @VioMNE 👍 Please check my comments. It would be nice if you can try using Template literals and format your code. Please choose do you want to use `""` or `''`. Good reading about return https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return 🌸

Thank you so much @cecastosic for great feedback. I appreciates it a lot :)

**Pull request merged**